projects/bugsplat-ng/package.json

```diff
@@ -12,6 +12,7 @@
   },
   "peerDependencies": {
     "@angular/common": ">=5.0.0",
-    "@angular/core": ">=5.0.0"
+    "@angular/core": ">=5.0.0",
+    "bugsplat": "^7.0.2"
```

Comment:
we need to test this in a real app to see if peerDeps is okay or if it needs to be a real dependency

Reply:
2cd1dca